### PR TITLE
python-six: Update to 1.12.0

### PR DESCRIPTION
**Pull request by neheb**
Switch to standard pythonhosted URL.

Cosmetic Makefile rearrangements for consistency between packages.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @jefferyto @commodo 
Compile tested: ramips

### CODE REVIEW

**Comment by jefferyto** [comment]
LGTM - thanks!

**Comment by commodo** [comment]
Rest looks good.
Thanks for this.
I was planning to allocate some time for this, in the next weeks. But I definitely don’t mind if others do it

Thanks 
Alex

**Comment by neheb** [comment]
> I would leave the project URL unchanged.
> If the project has a page on pypi, I feel it’s definitely preferred as it’s the canonical page for all Python package.

This URL is from PyPi. Most other projects here are like this.

File: lang/python/python-six/Makefile [hunk 1]
```diff
@@ -35,7 +34,7 @@ define Package/python-six/Default
   SECTION:=lang
   CATEGORY:=Languages
   SUBMENU:=Python
-  URL:=https://pypi.python.org/pypi/six
+  URL:=https://github.com/benjaminp/six
```
Comment (commodo) [inline lang/python/python-six/Makefile]: I mean this URL should be unchanged.
I was replying from my phone the first time, and the comment didn't seem to go on this line

Comment (jefferyto) [reply lang/python/python-six/Makefile]: It seems they have decided to set their GitHub page as their homepage (benjaminp/six#247), so I'm ok with this change.

**Comment by cshoredaniel** [comment]
@neheb  Has this been **run** tested?  Have apps that depend on it been tested?

**Comment by neheb** [comment]
This is currently installed on my router with no ill effects, so yes.

**Comment by cshoredaniel** [comment]
@neheb Ok, please include that in the PR message; it's important to know.  Although 'with no ill effects' -- do you use things depend on 'python-six'?  Because it's a library and doesn't actually do anything unless you're using a program that uses it (as I'm sure you are aware).

**Comment by neheb** [comment]
Yep. Seafile.

**Comment by cshoredaniel** [comment]
Ok, great!  Then please put that in commit message -- not saying run tested is something that makes me wonder if something is going to break on folks.

**Pull request merged**